packages/aeproject-cli/aeproject-init/init.js

```diff
@@ -59,8 +62,44 @@ const createAEprojectProjectStructure = async (shape) => {
     print('===== AEproject was successfully initialized! =====');
 }
 
+const compareSdkVersions = async (_sdkVersion, cwd) => {
+    // get current sdk version
+    let userPackageJson = JSON.parse(fs.readFileSync(path.join(cwd, './package.json'), 'utf8'));
+    let userSdkVersion = userPackageJson.dependencies['@aeternity/aepp-sdk'];
+
+    if (userSdkVersion) {
+        let userVersioning = userSdkVersion.split('.');
+        let updateToVersioning = _sdkVersion.split('.');
+
+        let promptMessage = `Found newer or different version of sdk ${ userSdkVersion }. Keep it, instead of ${ _sdkVersion }?`;
+
+        for (let i = 0; i < 3; i++) {
+            let user = userVersioning[i];
+            let updateTo = updateToVersioning[i];
+
+            if (!isNaN(updateTo) && !isNaN(user)) {
+                if (parseInt(user) > parseInt(updateTo)) {
```

Comment:
why comparing only portions of versions but not the whole version instead?

Reply:
how can we compare which of the version is higher 5.0.0 or 6.0.0